pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -179,8 +182,23 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
-	if _, err := pna.kubeClient.CoreV1().Services(pna.namespaceName).Create(context.Background(), hostNetworkTargetService, metav1.CreateOptions{}); err != nil {
```

Comment:
Where did this line go?

Reply:
whoops, removed accidentally from rebasing. fixed